baldrick/plugins/artifact_status.py

```diff
@@ -0,0 +1,46 @@
+import requests
+
+from .circleci import circleci_webhook_handler
+
+
+HOST = "https://api.github.com"
+
+
+@circleci_webhook_handler
+def set_commit_status_for_artifacts(repo_handler, payload, headers):
+    if payload['status'] == 'success':
+        artifacts = get_artifacts_from_build(payload)
+
+        urls = repo_handler.get_config_value("circleci_artifacts",
+                                             {"sphinx": {
+                                                 "url": "html/index.html",
+                                                 "message":
+                                                 "Click details to preview the documentation build"}})
```

Comment:
If this plugin is specifically for documentation, the plugin should probably be called something more specific than ``artifact_status``

Reply:
It defaults to documentation, but is now configurable for everything. (defaults for backwards compat)

Reply:
Ok cool

Reply:
Do we care about backward-compatibility since this is a new package though?...